song/app.js

```diff
@@ -32,13 +32,48 @@ app.get("/ping", function (req, res, next) {
   res.json({ message: "pong" });
 });
 
+app.get("/list", async (req, res, next) => {
+  await appDataSource.query(
+    `SELECT
+            users.id as userId,
+            users.profile_image as userProfileImage,
+            posts.user_id as postingId,
+            posts.image_url as postingImageUrl,
+            posts.content as postingContent
+            FROM users LEFT JOIN posts ON users.id = posts.user_id`
+  );
+  res.status(200).json({ data: rows });
+});
```

Comment:
1. 해당 API 잘 호출 되시나요? {data: rows}에서 rows가 어디에도 정의 되지 않아서 결과가 안 나올 것 같은데, 확인해 주세요.
2. 전체 Post를 호출하는 API 같은데, 이럴 경우에는 FROM을 users가 아닌 posts에서 시작하는 것이 좋습니다.
```suggestion
app.get("/list", async (req, res, next) => {
  const rows = await appDataSource.query(
    `SELECT
          users.id as userId,
          users.profile_image as userProfileImage,
          posts.user_id as postingId,
          posts.image_url as postingImageUrl,
          posts.content as postingContent
        FROM posts
        JOIN users ON posts.user_id = users.id`
  );
  res.status(200).json({ data: rows });
});
```